integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -806,7 +806,9 @@ public void callWebAppAndVerifyLoadBalancing(String webappName, boolean verifyLo
 
       LoggerHelper.getLocal().log(Level.INFO, "Curl cmd with response code " + curlCmdResCode);
       LoggerHelper.getLocal().log(Level.INFO, "Curl cmd " + curlCmd);
-
+      //check helm status
+      String lbName = getLoadBalancerName().toLowerCase();
+      TestUtils.checkHelmChart(lbName + "-operator", lbName);
```

Comment:
I think this is only for debugging..you may have to check the ingress as well ..

Reply:
added